DependencyInjection/Configuration.php

```diff
@@ -37,8 +37,7 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $treeBuilder->root('translation_tools');
+        $treeBuilder = new TreeBuilder('translation_tools');
```

Comment:
Nice catch @mickaelandrieu as it can be seen on https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php#L30

Reply:
Yes, and on https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php#L51

Reply:
This is not compatible with Symfony 3.4 though...